firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionEvent.kt

```diff
@@ -59,6 +59,41 @@ internal data class SessionInfo(
   /** What order this Session came in this run of the app. For the first Session this will be 0. */
   val sessionIndex: Int,
 
+  /** Tracks when the event was initiated */
+  var eventTimestampUs: Long,
+
+  /** Data collection status of the dependent product SDKs. */
+  var dataCollectionStatus: DataCollectionStatus = DataCollectionStatus(),
+
   /** Identifies a unique device+app installation: go/firebase-installations */
   var firebaseInstallationId: String = "",
 )
+
+/** Contains the data collection state for all dependent SDKs and sampling info */
+internal data class DataCollectionStatus(
+  val performance: DataCollectionState = DataCollectionState.COLLECTION_ENABLED,
+  val crashlytics: DataCollectionState = DataCollectionState.COLLECTION_ENABLED,
```

Comment:
Is the default `COLLECTION_ENABLED` (for now?)?

Reply:
Yeah I'm putting it like this so it looks right in our data but we'll need to change this when we integrate with the product SDKs